src/DGtal/topology/tables/LookUpTableFunctions.h

```diff
@@ -0,0 +1,188 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file LookUpTableFunctions.h
+ *
+ * @author Pablo Hernandez-Cerdan. Institute of Fundamental Sciences.
+ * Massey University. Palmerston North, New Zealand
+ *
+ * @date 2016/03/25
+ *
+ * Generation of Look Up Tables for predicates function in a voxel
+ * neighborhood.
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(LookUpTableFunctions_RECURSES)
+#error Recursive header files inclusion detected in LookUpTableFunctions.h
+#else // defined(LookUpTableFunctions_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define LookUpTableFunctions_RECURSES
+
+#if !defined LookUpTableFunctions_h
+/** Prevents repeated inclusion of headers. */
+#define LookUpTableFunctions_h
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include <bitset>
+#include <memory>
+#include "DGtal/topology/Object.h"
+#include "DGtal/helpers/StdDefs.h"
+#include <unordered_map>
+#include "boost/dynamic_bitset.hpp"
+
+namespace DGtal
+{
+  namespace lut {
```

Comment:
In a similar way, the filename ```LookUpTableFunctions``` is not great.. I have the feeling that it could be confusing. What do you think?